class Foo {
  Foo() throws Exception {}
}
